resources/views/rows.blade.php

```diff
@@ -21,11 +21,11 @@
                         <tbody>
                             @foreach ($users as $user)
                                 {{-- Task: only every second row should have "bg-red-100" --}}
-                                <tr class="bg-red-100">
-                                    <td>{{-- Task: add row number here: 1, 2, etc. --}}</td>
+                                <tr class="{{$loop->iteration % 2 == 0 ? 'bg-red-100' : ''}}">
```

Comment:
You can use the following keywords: even, odd, first

check it: https://laravel.com/docs/10.x/blade#the-loop-variable